controllers/component_dependency_update_controller_test.go

```diff
@@ -402,14 +403,301 @@ var _ = Describe("Component nudge controller", func() {
 				return renovatePipelinesCreated == 1 && renovateConfigsCreated == 2 && failureCount == 0
 			}, timeout, interval).WithTimeout(ensureTimeout).Should(BeTrue())
 		})
+
+		It("Test build performs nudge on success, only namespace wide renovate config provided", func() {
```

Comment:
There is 3 test cases with only small difference. Can we extract common part and avoid duplication?